src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const secondCell = rows[i].cells[1];
+  const clonedCell = secondCell.cloneNode(true);
+
+  rows[i].insertBefore(clonedCell, rows[i].cells[rows[i].cells.length - 1]);
```

Comment:
This line is a bit confusing. It's not immediately clear what you're trying to achieve. Consider adding a comment explaining what this line does.